### ISSUE TITLE
New check: unset readonly variables

### ISSUE THREAD

**Issue by felipecrs** [koalaman/shellcheck#2528]
#### For new checks and feature suggestions
- [x] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

readonly my_var=true
unset my_var
```

The above produces the following Bash error:

```
bash: unset: my_var: cannot unset: readonly variable
```

#### Here's what shellcheck currently says:

Nothing.

#### Here's what I wanted or expected to see:

Something like:

```
cannot unset readonly variables
```
